myintegrations/gitea-repositories/user1/repo1.git/hooks/post-receive.d/gitea

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+"/home/ethantkoenig/.gopath/src/code.gitea.io/gitea/gitea" hook --config='integrations/app.ini' post-receive
```

Comment:
I've updated the scripts to use `$GITEA_ROOT` instead of a hard-coded path. We'll have to export the appropriate value for `GITEA_ROOT` before running integration tests involving these scripts.